soh/soh/OTRGlobals.cpp

```diff
@@ -1645,6 +1645,8 @@ extern "C" int CustomMessage_RetrieveIfExists(PlayState* play) {
             }
         } else if (Randomizer_GetSettingValue(RSK_DAMPES_DIARY_HINT) && textId == TEXT_DAMPES_DIARY) {
             messageEntry = CustomMessageManager::Instance->RetrieveMessage(Randomizer::randoMiscHintsTableID, TEXT_DAMPES_DIARY);
+        } else if (Randomizer_GetSettingValue(RSK_RAINBOW_BRIDGE) == RO_BRIDGE_GREG && (textId == 0x704C || textId == 0x63 || textId == 0x84)) {
```

Comment:
Is `0x63` here supposed to be `0x6E` ? When comparing these numbers to the numbers in `z_en_takara_man`

Reply:
probably, i hadn't gotten to the in game testing part yet, good catch!